### PR TITLE
Annotations describing consumers producers

### PR DESCRIPTION
**Pull request by Felipe444**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,14 +7,46 @@
...
 ## Overview <a name="Overview"></a>
 
 This repository contains toolkit to define and validate contracts in JMS
 
+## Idea <a name="Idea"></a>
+
+There are 3 most typical scenarios and any other scenario is a combination of these 3.
+
+Scenario I:    Service 1 -> Queue A -> Service 2
+Scenario II:   Service 3 -> Topic B -> Service 4
```
Comment (matek2305) [inline README.md]: formatting here, each scenario should be in separate line

File: README.md [hunk 2]
```diff
@@ -7,14 +7,46 @@
...
+Scenario III:  Service 5 -> Topic C -> Queue D -> Service 6
+
+In Scenario I Service 1 sends commands to Queue A. Service 2 handles these commands.
```
Comment (matek2305) [inline README.md]: maybe change to "In first scenario ..."

**Review by matek2305 — CHANGES_REQUESTED** [review]
there are some issues with how README.md is displayed i think, missing line breaks etc., pleas check if it displayed how you wanted it

**Review by ndrwtrsk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Felipe444) [reply README.md]: done

**Review by matek2305 — APPROVED** [review]

**Pull request merged**